Tests/Marketplace/search_and_install_packs.py

```diff
@@ -591,18 +594,19 @@ def search_and_install_packs_and_their_dependencies_private(test_pack_path: str,
 
 
 def search_and_install_packs_and_their_dependencies(pack_ids: list,
-                                                    client: demisto_client):
+                                                    client: demisto_client, hostname: str = ''):
```

Comment:
why did we need this change? what happened before?

Reply:
The hostname in this function is: `client.host`.
I didnt liked this since the client's host in xsiam is the base_url that used only for API calls. 
UI url is different!! (I mean that if you click on the link that was printed before - the client.host, you will get blank page with 200 on it). 
This host is printed tooo much times in the logs, and it could create a lot of confusions for developers that will just click on the link and get blank page with 200 on it.